salt/modules/rh_ip.py

```diff
@@ -655,16 +654,16 @@ def _parse_settings_eth(opts, iface_type, enabled, iface):
             for opt in vlan:
                 result[opt] = opts[opt]
 
-    if iface_type not in ("bond", "team", "teamport", "vlan", "bridge", "ipip"):
+    if iface_type not in ("bond", "team", "vlan", "bridge", "ipip"):
         auto_addr = False
-        if "addr" in opts:
```

Comment:
That's a breaking change for existing SLS files.

Now I have no problem with doing that because it's aligning the config item HWADDR with the config key in salt, which is a good thing... But that's something that should definitely go into the changelog.